src/components/ArticleView/CommentEntry.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { postCommentByArticleId } from '../../../api';
+
+const CommentEntry = ({ article_id, onCommentPosted }) => {
+    const [userComment, setUserComment] = useState('');
+
+    function handlePostComment() {
+        postCommentByArticleId(article_id, {
+            username: 'cooljmessy',
+            body: userComment,
+        })
+        .then(() => {
+            setUserComment('');
+            onCommentPosted();
+        })
+        .catch(err => {
+            console.error('Failed to post comment:', err);
```

Comment:
You can come back to this on a later ticket, but what would you want to see as the user if your post failed?